.github/workflows/simulations.yml

```diff
@@ -7,8 +7,28 @@ on:
       - release/**
 
 jobs:
+  cleanup-runs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: rokroskar/workflow-run-cleanup-action@master
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/main'"
```

Comment:
this is deprecated, let's don't use it: https://github.com/rokroskar/workflow-run-cleanup-action

I have other PR to use the concurrency parameter, but didn't have time to inspect the errors, if you want you can look at it: https://github.com/umee-network/umee/pull/1669

Reply:
Ok